versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/spi/DatabaseAdapterUtil.java

```diff
@@ -47,14 +47,11 @@ public static Hasher newHasher() {
     return Hashing.sha256().newHasher();
   }
 
-  @SuppressWarnings("UnstableApiUsage")
   public static Hash randomHash() {
     ThreadLocalRandom rand = ThreadLocalRandom.current();
-    Hasher hasher = newHasher();
-    for (int i = 0; i < 20; i++) {
-      hasher.putInt(rand.nextInt());
-    }
-    return Hash.of(UnsafeByteOperations.unsafeWrap(hasher.hash().asBytes()));
```

Comment:
Will the random generator produce the same quality of dispersion as the hasher?

Could you do the "picture" test for old and new generators? (cf. https://en.wikipedia.org/wiki/Low-discrepancy_sequence#/media/File:Subrandom_2D.gif)

Reply:
Hm - good point.

Reply:
However, not sure how hashing would have helped to mitigate a bad PRNG ;)

Reply:
It could smooth out / break down clusters, I suppose.

Reply:
I'm going to try and generate pictures anyway :)

Reply:
I do not see any major dispersion difference:

Old hashed:
![image](https://user-images.githubusercontent.com/40603221/170387268-324183c3-b5fe-42aa-8927-6847a40d0a9f.png)

New random:
![image](https://user-images.githubusercontent.com/40603221/170387283-31f9e264-4625-4edc-87bb-6c54e3365188.png)

Reply:
... and just out of curiosity, here's the effect of hashing on a (artificially) poor quality random generator:

Just random:
![image](https://user-images.githubusercontent.com/40603221/170392212-57aba122-e26a-4016-8b16-5559c355418a.png)

same random generator, but hashed:
![image](https://user-images.githubusercontent.com/40603221/170392258-ce9ccffa-e9df-4d13-bc19-c715b4870807.png)